packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -163,6 +164,8 @@ const Injector = ({ children }: { children: React.ReactNode }): JSX.Element => {
   const [tokenBridgeParams, setTokenBridgeParams] =
     useState<TokenBridgeParams | null>(null)
 
+  useTransactionsInApp()
```

Comment:
the methods from `useTransactions` are used in many places of the code base
their methods are tied to the instance's `transactions` array and so it could only be instantiated once
otherwise the `transactions` array would be a fresh instance every time we call it

custom hooks don't necessarily need to return a value